lib/ddtrace/contrib/rails/abstract_controller/callbacks.rb

```diff
@@ -0,0 +1,16 @@
+require 'ddtrace/contrib/rails/abstract_controller/callbacks/action_tracing'
+
+module Datadog
+  module Contrib
+    module Rails
+      module AbstractController
+        # Includes correct Callbacks module
+        module Callbacks
+          def self.included(base)
+            base.prepend(ActionTracing) if ::Rails.version >= '5.0'
```

Comment:
We can provide this approach for different Rails versions. But from what you wrote in the description, this is a private API that can change during the time. In that case we **must** be sure that if a new Rails version is out, we're not going to break the application. Totally fine if we simply disable the functionality, since it would be a bug in a supported version and not a critical issue that forces users to disable the Tracer.

Reply:
I think I'll change this version check to patch only for versions 5.0.x and 5.1.x, nothing older, nothing newer. I think we have reasonable assurance that they will stay stable within those minor versions.